flixel/input/mouse/FlxMouse.hx

```diff
@@ -51,7 +51,7 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
 	 * sits at the right "height". Not used on flash with the native cursor API.
 	 */
 	public var cursorContainer(default, null):Sprite;
-	
+
```

Comment:
for future PRs set your hxformat.json to 
```json
"trailingWhitespace": true
```
and your settings.json should have
```json
"[haxe]": {
		"editor.formatOnSave": true,
		"editor.formatOnSaveMode": "modifications"
	},
	```

Reply:
Alright noted!